src/utilities/fetch-package-repos.mjs

```diff
@@ -33,17 +32,6 @@ const fetch = {
 };
 
 async function main() {
-  let api;
-  if (process.env.CI && process.env.CI === true) {
```

Comment:
`process.env.CI` is of type `string`, not `boolean`, so the original code was actually wrong which means the api wasn't authenticated at all before.